R/provider-cortex.R

```diff
@@ -163,7 +163,7 @@ method(chat_request, ProviderCortex) <- function(provider,
   req <- ellmer_req_credentials(req, provider@credentials)
   req <- req_retry(req, max_tries = 2)
   req <- ellmer_req_timeout(req, stream)
-  req <- req_user_agent(req, snowflake_user_agent())
+  req <- ellmer_req_user_agent(req, Sys.getenv("SF_PARTNER"))
```

Comment:
We might have to adjust this slightly more for Snowflake specifically. I know they're having issues with our existing user agents. But that probably belongs in a followup PR, not this one.